ui/pages/Project/components/RnaSeqResultPage.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Loader, Grid, Dropdown } from 'semantic-ui-react'
+
+import { getGenesById, getIndividualsByGuid, getRnaSeqDataByIndividual, getRnaSeqSignificantJunctionData } from 'redux/selectors'
+import { RNASEQ_JUNCTION_PADDING } from 'shared/utils/constants'
+import DataLoader from 'shared/components/DataLoader'
+import FamilyReads from 'shared/components/panel/family/FamilyReads'
+import RnaSeqJunctionOutliersTable from 'shared/components/table/RnaSeqJunctionOutliersTable'
```

Comment:
It is just a `DataTable` component, which must be loaded already. And it will be reused for the variant display. It is likely to be a required module. So it may save little with lazy import.